prov/verbs/src/fi_verbs.c

```diff
@@ -436,8 +436,12 @@ static int fi_ibv_param_define(const char *param_name, const char *param_str,
 		switch (type) {
 		case FI_PARAM_STRING:
 			if (*(char **)param_default != NULL) {
-				strncpy(param_default_str, *(char **)param_default, 256);
-				param_default_sz = strlen((char *)param_default_str);
+				param_default_sz =
+					MIN(strlen((char *)param_default_str),
+					    254);
```

Comment:
param_default_str is an empty string so strlen of it will always be 0.

Reply:
Thanks! The wrong string buffer were `strlen`'ed